lib/dredd_hooks/cli.rb

```diff
@@ -10,6 +10,7 @@ def self.start(error=STDERR, out=STDOUT, files)
 
       # Run the server
       out.puts 'Starting Ruby Dredd Hooks Worker...'
+      out.flush
```

Comment:
I'm guessing this relates to [waiting for the server to output "Starting"](https://github.com/apiaryio/dredd-hooks-ruby/pull/37/files#diff-60124dd45a23a8b2ea2358d2c420d09aR5). Ensuring the output seems reasonable to me. :white_check_mark:

Reply:
Exactly 👍